dashdataframe/__init__.py

```diff
@@ -48,104 +102,48 @@ def configure_app(app, df,
         what function to call if you make a link
 
     """
+
     if plot_columns is None:
+        
         plot_columns = df.select_dtypes(np.number).columns
 
+
     color_options = list(plot_columns)
     assert np.all(np.isin(np.array(plot_columns), df.columns))
     color_options.append('None')
+ 
+    scatter_layout = dfl.make_scatter_layout(plot_columns, color_options, link_name)
+    scatter_tab = dcc.Tab(label='Scatter Plot Explorer', children=[scatter_layout])
+    tabs = [scatter_tab]
```

Comment:
I wonder if the class should have a 'add_tab' method.. feel free to debate.  this would let people extend it on their own?